gitlab/packages/gitlab/gitlab_create.js

```diff
@@ -0,0 +1,65 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+
+// Sends ticket to gitLab API
+
+const axios = require('axios');
+
+async function sendIssue(url, issue) {
+  return await axios.post(url, issue)
+    .then(response => { return response.data; })
+    .catch(error => { return error.response.data; });
+}
+
+async function _command(params, commandText, secrets = {}) {
+  
+  const {
+    repo,
+    title,
+    description
+  } = params;
+  
+  const issue = {
+    title: title,
+    description: description,
+    access_token: secrets.AcessToken_GitLab,

```

Comment:
```suggestion
    access_token: secrets.gitlabToken,
```

Reply:
Rename `AcessToken_GitLab` to `gitlabToken` wherever required.

Reply:
Can do 👍